src/plugin.ts

```diff
@@ -38,6 +36,6 @@ function activateWidgetExtension(
   registry.registerWidget({
     name: MODULE_NAME,
     version: MODULE_VERSION,
-    exports: widgetExports,
+    exports: () => import('./widget'),
```

Comment:
```suggestion
    exports: () => import('./index'),
```

we also import from `graph` and importing from index picks up both